monitor/backends_stat_provider.cpp

```diff
@@ -188,6 +216,10 @@ static rapidjson::Value& backend_stats_json(uint64_t categories,
 		fill_disabled_backend_config(stat_value, allocator, config_backend);
 	}
 
+	if (categories & DNET_MONITOR_BACKEND) {
```

Comment:
This has to be moved into fill_disabled_backend_config() function, since if there is no json generated, there will be no ["backend"]["config"] section, and it will crash